pkg/test/test.go

```
@@ -75,6 +75,9 @@ func RunTests(t *testing.T, suite TestingSuite, config *Config) {
 
 	methodFinder := reflect.TypeOf(suite)
 	tests := []testing.InternalTest{}
+
+	checkTestName(t, config, methodFinder)
```

Perhaps you can return the index or method here to use it in the next few lines. In addition, you can return an error and perform the failure function here.

---

There is no method to return when the test name is "", which is the case when all tests are to be run

---

OK.